20180908/jobCentTelegramBot/sponsor/server/controllers/sponsors.js

```diff
@@ -0,0 +1,56 @@
+const Sponsor = require("../models").Sponsor;
+const nCentSDK = require("ncent-sandbox-sdk");
+const nCentSDKInstance = new nCentSDK();
+
+module.exports = {
+  create(req, res) {
```

Comment:
We should comment what each of these variables are and what type we are expecting. We should do some check to make sure people are sharing valid types